airflow/providers/amazon/aws/triggers/emr.py

```diff
@@ -461,4 +460,9 @@ def __init__(
         )
 
     def hook(self) -> AwsGenericHook:
-        return self.hook_instance
+        return EmrServerlessHook(self.aws_conn_id)
+
+    @property
+    def hook_instance(self) -> AwsGenericHook:
+        """This property is added for backward compatibility."""
+        return self.hook()
```

Comment:
I added `hook_instance` as a property, so it will be available when the user tries to access it, but it's lazy evaluated and created only when accessing it.

That makes the change backward compatible.